src/explore-education-statistics-admin/src/utils/ckeditor/getMarkersOrder.ts

```diff
@@ -0,0 +1,22 @@
+import { Marker } from '@admin/types/ckeditor';
+
+// Get the order of the markers based on their position in the editor.
+// Used to order the comments list.
+const getMarkersOrder = (markers: Marker[]) => {
```

Comment:
Would declare the return type to be explicit - seems like it should be `string[]`?